icons/hostelworld.svg

```diff
@@ -0,0 +1,3 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+  <title>Hostelworld</title>
+  <path d="M3.294.015c-1.85 0-3.34 1.49-3.34 3.34v17.32c0 1.85 1.49 3.34 3.34 3.34h17.32c1.85 0 3.34-1.49 3.34-3.34V3.354c0-1.85-1.49-3.34-3.34-3.34zm.822 3.803h4.805a.34.34 0 0 1 .334.334v4.291h-.54c-.05 0-.076.026-.102.051a.156.156 0 0 0 0 .206l3.084 3.11a.31.31 0 0 1 0 .41l-3.058 3.11a.2.2 0 0 0-.052.102c-.025.077.052.155.129.155h.54v4.265a.34.34 0 0 1-.335.334H4.116a.34.34 0 0 1-.334-.334v-15.7a.34.34 0 0 1 .334-.334m10.895.025h4.805c.18 0 .309.13.36.309v15.726a.34.34 0 0 1-.334.334h-4.805a.34.34 0 0 1-.334-.334v-4.291h.54c.077-.026.128-.078.128-.155 0-.025 0-.051-.025-.077l-3.084-3.135a.31.31 0 0 1 0-.41L15.32 8.7c.026-.026.026-.052.026-.077 0-.077-.052-.154-.13-.154h-.539V4.177a.34.34 0 0 1 .334-.334" style="fill:#000;fill-opacity:1;stroke:none;stroke-width:.25"/></svg>
```

Comment:
This isn't optimized or formatted as per our guidelines: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon.